bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.html

```diff
@@ -184,11 +184,15 @@ <h2>Webhook configuration</h2>
     </div>
     <ng-template #sendStartedOverlay>
       <p>
-        If you subscribe your webhook to a task event of type triggered, the corresponding started and finished events
-        are sent automatically.
+        Sending started and finished from webhook service is an "async" model. The Keptn task immediately returns. Async
+        is easier at first to implement as it needs no changes to the receiving side, but the task timing does not
+        include the processing time of the receiving system. If you subscribe your webhook to a task event of type
+        triggered, the corresponding started and finished events are sent automatically.
```

Comment:
```suggestion
        ` triggered`, the corresponding started and finished events are sent automatically.
```

Reply:
Thanks @StackScribe ! I just made changes without noticing you. Could you review again? 😄